packages/amino/src/encoding.ts

```diff
@@ -61,6 +62,9 @@ export function decodeAminoPubkey(data: Uint8Array): Pubkey {
       type: pubkeyType.sr25519,
       value: toBase64(rest),
     };
+  } else if (arrayContentStartsWith(data, pubkeyAminoPrefixMultisigThreshold)) {
+    // eslint-disable-next-line @typescript-eslint/no-use-before-define
+    return decodeMultisigPubkey(Array.from(data));
```

Comment:
Please move implementation up such that we don't need to disable this lint. I know this is typically done different in many other code bases but here we decided to keep the link and have the dependencies above.

Reply:
Here is a problem. These two functions invoke each other. Thus we if a place one on top, another will produce a warning. That's why I suppressed the warning.

Reply:
Also, I read SDK code, and participants of multisig can be another multisig. So we have some sort of recursion here.

Reply:
Ah, I see, thanks for clarification!